app-misc/figlet/figlet-222.recipe

```diff
@@ -9,24 +9,39 @@ COPYRIGHT="1991, 1993, 1994 Glenn Chappell and Ian Chai
 	2002 Christiaan Keet"
 LICENSE="AFL v2.1"
 REVISION="1"
-SOURCE_URI="ftp://ftp.figlet.org/pub/figlet/program/unix/figlet222.tar.gz"
```

Comment:
Looks like the current version of figlet is 2.2.5, please update this to the latest release.